client/signup/steps/creds-confirm/index.jsx

```diff
@@ -0,0 +1,79 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import Card from 'components/card';
+import Button from 'components/button';
+import { autoConfigCredentials } from 'state/jetpack/credentials/actions';
+
+class CredsConfirmStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+	};
+
+	autoConfigCredentials = () =>
+		this.props.autoConfigCredentials( this.props.initialContext.query.blogid );
+
+	skipStep = () => this.props.goToNextStep();
+
+	shareCredentials = () => {
+		this.autoConfigCredentials();
+		this.props.goToStep( 'creds-complete' );
+	};
+
+	contactSupport = () => {
+		// live chat?
+	};
+
+	renderStepContent = () => {
+		const { translate } = this.props;
+
+		return (
+			<Card className="creds-confirm__card">
+				<h3 className="creds-confirm__title">{ translate( 'Are you sure?' ) }</h3>
+				<img className="creds-confirm__image" src="/calypso/images/illustrations/security.svg" />
+				<p className="creds-confirm__description">
+					{ translate(
+						"If you don't share credentials with Jetpack, your site won't be backed up. Our " +
+							'support staff is available to answer any questions you might have.'
+					) }
+				</p>
+				<Button onClick={ this.contactSupport }>{ translate( 'Contact support' ) }</Button>
```

Comment:
Hi! I've found a possible matching string that has already been translated **34** times:
`translate( 'Contact Support' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/pressable-nux-creds)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).